### PR TITLE
Add migration guide for new icons and update contribution guide

### PR DESCRIPTION
**Pull request by Faradey27**
### What changed

...

### Why it changed

...

---

- [ ] Change is tested
- [ ] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by Faradey27** [comment]
@shlomitc let discuss this tomorrow

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -77,5 +77,12 @@ open localhost:6006
...
+* go to [wix-ui](https://github.com/wix/wix-ui) repo to [wix-ui-icons-common](https://github.com/wix/wix-ui/tree/master/packages/wix-ui-icons-common) package
+* add it to the **system** icons folder if it's an icon used internally inside a wix-style-react component, or to the **general** in case it is an icon to be used inside an app
+* add your icon to correct package folder inside [wix-ui-icons-common](https://github.com/wix/wix-ui/tree/master/packages/wix-ui-icons-common)
+* create pull request for [wix-ui](https://github.com/wix/wix-ui) with your new icon
+* after merging your pull request inside [wix-ui](https://github.com/wix/wix-ui), you need to bump and use the right version of wix-ui-icons-common, then your new icon will appear inside [wix-style-react](https://github.com/wix/wix-style-react) with next wix-style-react release(wix-style-react has automation script, which take all icons from [wix-ui-icons-common](https://github.com/wix/wix-ui/tree/master/packages/wix-ui-icons-common) and proxy them inside [wix-style-react](https://github.com/wix/wix-style-react))
```
Comment (shlomitc) [inline docs/CONTRIBUTING.md]: simplify it a little bit.
In `wix-style-react`'s package.json - update `wix-ui-icons-common`'s version to the new one.

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -77,5 +77,12 @@ open localhost:6006
...
+* your icon will be available in wix-style-react with such syntax `import MyAwesomeIcon from 'wix-style-react/new-icons/MyAwesomeIcon'` or `import MyAwesomeIcon from 'wix-style-react/new-icons/system/MyAwesomeIcon'`
```
Comment (shlomitc) [inline docs/CONTRIBUTING.md]: the new icon wil be...

File: docs/NEW_ICONS_MIGRATION.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+First of all, some names are changed and some icons are deprecated, so please visit http://electric-process.surge.sh/ and find new icon name, if your icon is deprecated - ask your UX/UI person to provide alternative from available icons.
+
+Now you have two choices:
+1) Do migration by hands based on mapping
+2) run our migration script
+```
+  npx migrate-to-icons-v2
+```
+such command will migrate all .js files inside src/ folder to new icons,
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: this command

File: docs/NEW_ICONS_MIGRATION.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# How to migrate to new icons
+
+First of all, some names are changed and some icons are deprecated, so please visit http://electric-process.surge.sh/ and find new icon name, if your icon is deprecated - ask your UX/UI person to provide alternative from available icons.
+
+Now you have two choices:
+1) Do migration by hands based on mapping
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: do the migration manually based on the mapping

File: docs/NEW_ICONS_MIGRATION.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+2) run our migration script
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: run the migration script. (please note, it currently works only for `.js` files)

File: docs/NEW_ICONS_MIGRATION.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+```
+  npx migrate-to-icons-v2
+```
+such command will migrate all .js files inside src/ folder to new icons,
+if you want to migrate icons in different folder, you can use --path param
+```
+  npx migrate-to-icons-v2 --path stories/
+```
+
+# Motivation
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: Why did we create new icons

File: docs/NEW_ICONS_MIGRATION.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+* current icons do not have standard sizes
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: The old icons had different sizes.

File: docs/NEW_ICONS_MIGRATION.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+* a lot of current icons are broken
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: Some icons were broken

File: docs/NEW_ICONS_MIGRATION.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+* it is tricky to import and to use current icons
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: mmm, not sure about it

File: docs/NEW_ICONS_MIGRATION.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+  import Add from 'wix-style-react/new-icons/Add';
+
+  export default () => (
+    <div>
+      <Add />
+    </div>
+  )
+```
+
+Example 2
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: I think we should remove it until we'll support tree shaking. I don't see a reason to suggest it now

File: docs/NEW_ICONS_MIGRATION.md (hunk 9 above)
Comment (Faradey27) [reply docs/NEW_ICONS_MIGRATION.md]: @shlomitc 
old way
```js
import Add from 'wix-style-react/dist/src/Icons/dist/components/Add';
```js
new way
```
import Add from 'wix-style-react/new-icons/Add';
```

**Comment by Faradey27** [comment]
@shlomitc updated

File: docs/NEW_ICONS_MIGRATION.md (hunk 9 above)
Comment (shlomitc) [reply docs/NEW_ICONS_MIGRATION.md]: I see. so write it as "the import path was too specific to the folder structure"

File: docs/NEW_ICONS_MIGRATION.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
+# How to migrate to new icons
+
+First of all, some names are changed and some icons are deprecated, so please visit http://electric-process.surge.sh/ and find new icon name, if your icon is deprecated - ask your UX/UI person to provide alternative from available icons.
+
+Now you have two choices:
+1) Do the migration manually based on the mapping
+2) run the migration script. (please note, it currently works only for .js files)
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: capital R

File: docs/NEW_ICONS_MIGRATION.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+# Why did we create new icons
+
+* the old icons had different sizes.
+* some icons were broken
+* it is tricky to import and to use current icons
+
+# How to use new icons?
+
+Example 1
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: remove title

File: docs/NEW_ICONS_MIGRATION.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
+# How to migrate to new icons
```
Comment (shlomitc) [inline docs/NEW_ICONS_MIGRATION.md]: Small teak to the structure (not everything should be a single `#`)
```
# New icons in `wix-style-react`

## How to migrate to the new icons

## Why did we create new icons

## Using the new icons in your code
```

**Pull request merged**